src/Vulkan/RHI/VkPipelineState.cpp

```diff
@@ -67,6 +67,8 @@ void VkGraphicsPipelineState::Compile(const Shader& vertexShader,
     {
         const VertexInputLayout::VertexAttribute& attribute = key.vertexInputLayout.attributes[i];
         // TODO: Find out the right location according to semanticName + semanticIndex from shader reflection
+        // Trello:
```

Comment:
Not to be too annoying but fyi you can get a shortened URL to the Trello ticket by using the share button after having clicked on a ticket